cudamapper/src/main.cu

```diff
@@ -275,13 +283,15 @@ void process_one_device_batch(const IndexBatch& device_batch,
 /// \param host_cache data will be loaded into cache within the function
 /// \param device_cache data will be loaded into cache within the function
 /// \param overlaps_and_cigars_to_process overlaps and cigars are output to this structure and the then consumed by another thread
+/// \param number_of_skipped_pairs_of_indices
```

Comment:
Can we note that indices are skipped due to oom_exceptions being thrown?

Reply:
done